dist/stat.py

```diff
@@ -159,7 +159,7 @@ def print_defines():
 f.close()
 compare_srcfile(tmp_file, '../src/include/wiredtiger.in')
 
-def print_func(name, handle, statlist):
+def print_func(name, capname, handle, statlist):
```

Comment:
I'm curious what `cap` stands for here: "capture"?
Like Don I find the invocation `print_func('session', '', None, session_stats)` confusing.

Please consider moving `capname` to the last arg, and giving it a default value (perhaps `None`), this way the session invocation would become:
```
print_func('session', None, session_stats)
```

Reply:
Hi Marc, capname stands for capital versions of the statistics names - e.g CONN, DSRC, JOIN etc. This is used elsewhere in the code so I am just reusing the naming here.

Reply:
Applied changes here [d90af1d](https://github.com/wiredtiger/wiredtiger/pull/10476/commits/d90af1d7976eebe351364c8df2edee4a299ee7ee)